docs/implementations.rst

```diff
@@ -48,6 +48,11 @@ Class: RISEScoring
 .. autoclass:: xaitk_saliency.impls.gen_classifier_conf_sal.rise_scoring.RISEScoring
   :members:
 
+Class: PolicyScoring
```

Comment:
I think we decided to list implementations in alphabetical order, so please adjust this order appropriately based on whatever final name for the implementation we decide upon.